### PR TITLE
converting terraform generic iam rules to hcl

### PR DESCRIPTION
**Pull request by colleend**
Converting so that these work in HCL!

### CODE REVIEW

File: terraform/lang/security/iam/no-iam-data-exfiltration.yaml [hunk 1]
```diff
@@ -61,8 +74,7 @@ rules:
     - https://cloudsplaining.readthedocs.io/en/latest/glossary/data-exfiltration/
     category: security
     license: Commons Clause License Condition v1.0[LGPL-2.1-only]
-  paths:
-    include:
-    - '*.tf'
-  languages: [generic]
+    technology:
```
Comment (inkz) [inline terraform/lang/security/iam/no-iam-data-exfiltration.yaml:78]: is it possible to add CWE number here?
(I would propose smth like https://cwe.mitre.org/data/definitions/200.html, but you know better)

**Review by inkz — CHANGES_REQUESTED** [review]
add CWE numbers to some rules (e.g. `no-iam-data-exfiltration`, `no-iam-priv-esc-funcs`)

if it actually make sense to do it :v:, because rules are IaC related maybe it is not applicable

**Review by aryx — COMMENTED** [review]
Woohoo!

**Comment by colleend** [comment]
sounds good, I can look into it and add the appropriate CWE #s to some of the rules!

**Review by minusworld — APPROVED** [review]

**Review by inkz — APPROVED** [review]
lgtm!

**Pull request merged**